### PR TITLE
Add get_bits() definition

### PR DESCRIPTION
**Pull request by JeromeMartinez**
There was no definition of this function.

Issue reported by @dwbuiten

### CODE REVIEW

File: ffv1.md [hunk 1]
```diff
@@ -172,6 +172,10 @@ Several components of FFV1 are described in this document using pseudo-code. Not
 
 `byte_aligned( )` is true if `remaining_bits_in_bitstream( NumBytes )` is a multiple of 8, otherwise false.
 
+#### get_bits
+
+`get_bits( i )` is the action to read the next `i` bits in the bitstream and to return the corresponding value. The current position is increased by `i`.
```
Comment (dericed) [inline ffv1.md]: nit: better term for `current position`?

Comment (retokromer) [reply ffv1.md]: I guess, I would use the term `pointer`.

Comment (JeromeMartinez) [reply ffv1.md]: I reused `current position` from line 169.
Should I change the term in the same PR?

**Comment by JeromeMartinez** [comment]
added a commit for changing "current position" to "pointer"

File: ffv1.md (hunk 1 above)
Comment (dericed) [reply ffv1.md]: I preferred the term `current position` over `pointer` but ¯\\_(ツ)_/¯

Comment (JeromeMartinez) [reply ffv1.md]: blank vote, need another voter.

**Comment by michaelni** [comment]
This definition seems unclear in relation to lest significant bit vs most significant bit in a byte first.

**Comment by JeromeMartinez** [comment]
PR updated with addition of `from most significant bit to least significant bit`

**Pull request closed without merging**